### PR TITLE
[NUI] Add WebView FeedMouseWheel as hidden api

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
[NUI] Add WebView FeedMouseWheel as hidden api

- should be released with the following dali patches:
[Tizen] Add WebView FeedMouseWheel as hidden api
https://review.tizen.org/gerrit/c/platform/core/uifw/dali-adaptor/+/316591
https://review.tizen.org/gerrit/c/platform/core/uifw/dali-toolkit/+/316592
https://review.tizen.org/gerrit/c/platform/core/uifw/dali-extension/+/316593
https://review.tizen.org/gerrit/c/platform/core/uifw/dali-csharp-binder/+/316594

- Add some logs in NUICoreBackend.Exit() by request from GBM

### API Changes ###
nothing

### CODE REVIEW

File: src/Tizen.NUI/src/internal/Application/NUICoreBackend.cs [hunk 1]
```diff
@@ -142,8 +142,10 @@ public void Dispose()
         /// </summary>
         public void Exit()
         {
+            Tizen.Log.Error("NUI", "NUICoreBackend.Exit() !");
```
Comment (JoogabYun) [inline src/Tizen.NUI/src/internal/Application/NUICoreBackend.cs:145]: Is the error log level correct?

Comment (dongsug-song) [reply src/Tizen.NUI/src/internal/Application/NUICoreBackend.cs:145]: Yes, you are correct. However, on the GBM TV target, logs at the error and fatal levels are visible. Due to the request from the GBM user, I had to include debugging logs at the error and fatal levels, although it was not ideal.

Comment (Seoyeon2Kim) [reply src/Tizen.NUI/src/internal/Application/NUICoreBackend.cs:145]: How about adding some comments on why these logs are `Error` for the history?

Comment (dongsug-song) [reply src/Tizen.NUI/src/internal/Application/NUICoreBackend.cs:148]: thank you for the review.
I have added some comments by following your review.

**Review by taehyub — APPROVED** [review]
LGTM

File: src/Tizen.NUI/src/internal/Application/NUICoreBackend.cs (hunk 1 above)
Comment (Seoyeon2Kim) [reply src/Tizen.NUI/src/internal/Application/NUICoreBackend.cs:148]: Thanks for updating it ;)

**Pull request merged**